LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -61,7 +61,7 @@ public struct InsulinModelSelection: View, HorizontalSizeClassOverride {
 
     let chartManager: ChartsManager = {
         let chartManager = ChartsManager(
-            colors: .default,
+            colors: ChartColorPalette.defaultPalette,
```

Comment:
nit: may want to expose this so users of InsulinModelSelection can control it, but the current use works.